lib/bolt/config.rb

```diff
@@ -46,6 +54,14 @@ def initialize(**kwargs)
             self[:transports][transport][k] = v
           end
         end
+
+        TRANSPORT_SPECIFIC_DEFAULTS.each do |k,v|
```

Comment:
This should be `TRANSPORT_SPECIFIC_DEFAULTS[transport].each`

Reply:
updated